pkg/testrunner/result/output.go

```diff
@@ -118,6 +108,13 @@ func getTestrunSummary(tr *tmv1beta1.Testrun, metadata *testrunner.Metadata) ([]
 				StartTime: step.StartTime,
 				Duration:  step.Duration,
 			}
+			if config.KibanaEndpoint != "" && tr.Status.Workflow != "" && summary.Name != "" {
```

Comment:
Maybe include all/some checks in the buildKibanaLogURL function to keep the getTestrunSummary lean?

Reply:
Makes absolutely sense, looks much better now, thanks for the hint!

Reply:
looks good now:)